agriculture_shop/demo/payment_provider_demo_post.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
```

Comment:
Probably better in noupdate, since we don't want to force the installation everytime